Jakefile.js

```diff
@@ -145,11 +145,12 @@ var harnessSources = [
 var librarySourceMap = [
         { target: "lib.core.d.ts", sources: ["core.d.ts"] },
         { target: "lib.dom.d.ts", sources: ["importcore.d.ts", "extensions.d.ts", "intl.d.ts", "dom.generated.d.ts"], },
+        { target: "lib.dom.es6.d.ts", sources: ["importcore.d.ts", "es6.d.ts", "intl.d.ts", "dom.generated.d.ts", "dom.es6.d.ts"] },

```

Comment:
i do not think you need this. the change to add the extensions to lib.es6.d.ts should be sufficient.